src/web/components/AdSlot.tsx

```diff
@@ -362,7 +379,7 @@ export const AdSlot: React.FC<Props> = ({ position, display }) => {
 							'ad-slot--mpu-banner-ad',
 							'ad-slot--rendered',
 						].join(' ')}
-						css={[labelStyles, fluidAdStyles, adSlotAboveNav]}
+						css={[adSlotAboveNav, adStyles, fluidFullWidthAdStyles]}
```

Comment:
Should `adSlotAboveNav` come after `adStyles` so it overrides it?

Reply:
Good catch!